src/server/license/cmds/cmds.go

```diff
@@ -253,7 +253,7 @@ func GetStateCmd() *cobra.Command {
 				fmt.Println("No Pachyderm Enterprise license is configured")
 				return nil
 			}
-			ts, err := types.TimestampFromProto(resp.Info.Expires)
+			ts, err := types.TimestampFromProto(resp.GetInfo().GetExpires())
```

Comment:
I think this still suffers from the same risk, since GetInfo() can return nil right?

Reply:
`GetExpires` also protects against `nil`:

```go
func (m *TokenInfo) GetExpires() *types.Timestamp {
	if m != nil {
		return m.Expires
	}
	return nil
}
```

And so does `TimestampFromProto`, so we should be good to go.

Reply:
Misread the autogenerated code, LGTM